### PR TITLE
Branch add customer tests

### LINKED ISSUE #80: Implement Customer Tests for parser

### PR DESCRIPTION
**Pull request by Jsjk112344**
Fixes #80

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -19,4 +19,12 @@ public class CliSyntax {
     public static final Prefix PREFIX_SHIFT_TIMING = new Prefix("shift/"); // Shift timing
```
Comment (findingfaey) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:19]: Consider adding a brief description here explaining what Prefix class does and why these particular prefixes are being used. It would help new developers understand the context better.

File: src/test/java/seedu/address/logic/commands/AddStaffCommandTest.java [hunk 2]
```diff
@@ -176,6 +177,36 @@ public void deletePerson(Person target) {
         public void addPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public ObservableList<Customer> getFilteredCustomerList() {
```
Comment (findingfaey) [inline src/test/java/seedu/address/logic/commands/AddStaffCommandTest.java:182]: Consider adding test cases to cover edge scenarios for this method, such as what happens when the customer list is empty or contains duplicate entries.
